docs/content/tutorial/step_02.ngdoc

```diff
@@ -49,128 +50,150 @@ __`app/index.html`:__
 </html>
 ```
 
-We replaced the hard-coded phone list with the {@link ng.directive:ngRepeat ngRepeat directive}
-and two {@link guide/expression Angular expressions}:
+We replaced the hard-coded phone list with the {@link ng.directive:ngRepeat ngRepeat} directive and
+two {@link guide/expression Angular expressions}:
```

Comment:
You should be able to simply write `{@link ngRepeat}` here, unless you don't want the `ngRepeat` to be styled as code, in which case you can write `{@link ngRepeat ntRepeat}`